contactos.php

```diff
@@ -1,22 +1,10 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dungeon's Home</title>
-    <link id="estilo" rel="stylesheet" type="text/css" href="csses/style.css" />
-</head>
-
-<body>
-    <header>Dungeon's Hub <img src="img/Dungeon's_Hub.png" height="128" width= "128"></header>
-    <footer>
-        <ul><li><a href="index.html">Inicio</a></li>
-            <li><a href="detalles.html">Detalles</a></li>
-            <li><a href="bocetos.html">Bocetos</a></li>
-            <li><a href="Miembros.html">Miembros</a></li>
-            <li><a href="planificacion.html">Planificacion</a></li>
-        </ul>
-    </footer>
+<?php
```

Comment:
Si queréis mantener el formulario de contacto ya toca implementarlo bien con PHP. Además podéis utilizar la función mail de PHP y consultar los mensajes que envieis en mail.containers.fdi.ucm.es